rebar.config

```diff
@@ -67,10 +67,6 @@
             {bootstrap, []},
 
             {prod, [
-                {escript_incl_extra, [
-                    {"relx/priv/templates/*", "_build/prod/lib/"},
-                    {"rebar/priv/templates/*", "_build/prod/lib/"}
-                ]},
```

Comment:
As a bonus, these overrides for the `prod` profile can be dropped since the new implementation computes the output directory (using the current profiles in effect).